### PR TITLE
Update to SDL 2.0.14

### PR DESCRIPTION
**Pull request by Perksey**
- [x] Updated submodule
- [x] Updated bindings
- [x] Updated native packages

### CODE REVIEW

File: src/Windowing/Silk.NET.SDL/SDL.cs [hunk 1]
```diff
@@ -15,16 +15,6 @@ namespace Silk.NET.SDL
...
-        public const uint InitJoystick = 0x00000200;
-        public const uint InitHaptic = 0x00001000;
-        public const uint InitGamecontroller = 0x00002000;
-        public const uint InitEvents = 0x00004000;
-        public const uint InitSensor = 0x00008000;
-        public const uint InitNoparachute = 0x00100000;
-
         public const uint InitEverything = InitTimer | InitAudio | InitVideo |
                                            InitEvents | InitJoystick | InitHaptic |
                                            InitGamecontroller | InitSensor;
```
Comment (HurricanKai) [inline src/Windowing/Silk.NET.SDL/SDL.cs:20]: um? have these moved to generated files?

**Review by HurricanKai — COMMENTED** [review]
Almost looks good - I think this also includes the SDL nint change & m64?

File: src/Windowing/Silk.NET.SDL/SDL.cs (hunk 1 above)
Comment (Perksey) [reply src/Windowing/Silk.NET.SDL/SDL.cs:20]: Yup.

**Comment by Perksey** [comment]
Yeah. D3D9 and SDL are now 64-bit only due to ABI deviations we can't efficiently account for. In addition, SDL hasn't been updated to fix the massively breaking nint bug fix we had in rtm (or 2.1 i forget)

**Comment by HurricanKai** [comment]
Awesome - don't forget about the GetWindowWMInfo thing, and then mark as ready for review, looks good to me then.

**Comment by Perksey** [comment]
Ready to go.

**Review by HurricanKai — APPROVED** [review]

**Pull request merged**